toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -1,4 +1,52 @@
 {
+  "border": {
+    "color": {
+      "default": {
```

Comment:
I left in the "default" name for now even though it sounds like we might not want that long term. If/when we decide on that we can have a separate task to handle this in a different way.

Reply:
Unfortunately it seems like we still need this even after switching to `value` as an object because we still have the same problem where `style-dictionary` doesn't parse token names like `text-color-deemphasized` if it's nested under `text-color` 😞